Dockerfile

```diff
@@ -10,7 +10,8 @@ COPY . .
 RUN make
 RUN touch /config.yaml
 
-FROM gcr.io/distroless/static
+FROM alpine:latest
```

Comment:
Thoughts on using the debug image of distroless instead to be consistent? (or should we consider switching everything back to alpine?) https://github.com/GoogleContainerTools/distroless#debug-images

Reply:
Oh, yes.  That's a better idea.

Reply:
Actually...there doesn't appear to be a debug image for static 🤨 

Alpine is very small surface area but distroless is certainly better from a security perspective.  I'm okay with the split here.  I'm going to save this comment so that I can hate myself in three months when we change them all to alpine 🙃

Reply:
Totally okay with the split.  But if you wanted to avoid it we could run `base-debug which adds a few extra clibs we don't need.

> Most other applications (and Go apps that require libc/cgo) should start with gcr.io/distroless/base, which contains all of the packages in gcr.io/distroless/static, and glibc, libssl, openssl

see: https://github.com/GoogleContainerTools/distroless/tree/master/base#documentation-for-gcriodistrolessbase-and-gcriodistrolessstatic

Reply:
Ah that makes sense.  Didn't dig enough.  Will change to that.